lib.ftd

```diff
@@ -139,8 +139,47 @@ ftd.image-src src:
 boolean $hover-me: false
 
 -- ftd.column:
-width.fixed.percent if { ftd.device != "mobile" }: 49
-width.fixed.percent if { ftd.device == "mobile" }: 100
+width: fill-container
+
+-- ftd.column:
+if: { ftd.device != "mobile" }
+width.fixed.percent: 49
+;;width.fixed.percent if { ftd.device == "mobile" }: 100
```

Comment:
Remove commented headers

Reply:
@Heulitig as per our `ftd` component building standards we should not pass CSS to component and make conditional on invocation.